bindings/go/evmc/host.c

```diff
@@ -0,0 +1,22 @@
+#include "_cgo_export.h"
+
+#include <stdlib.h>
+
+const struct evmc_context_fn_table fn_table = {
```

Comment:
I'm not sure how linking works and whether this global symbol would pollute any consumer of this go library, but probably it makes sense prefixing these.

Reply:
Yes. I agree. I can also add "hidden" attribute to it.